apps/next-next-app/next.config.js

```diff
@@ -0,0 +1,5 @@
+const withTM = require('next-transpile-modules')(['ui'])
```

Comment:
Which version of nextjs are you using. This package is now baked into the Next config. 
- [Refer to this](https://github.com/martpie/next-transpile-modules/issues/280#issuecomment-1292615030)
- [transpilePackages](https://nextjs.org/docs/app/api-reference/next-config-js/transpilePackages)

Reply:
updated